grakn-test/src/test/java/ai/grakn/test/graql/analytics/ScalingTestIT.java

```diff
@@ -301,7 +304,8 @@ public void testLargeDegreeMutationResultsInReadableGraphIT() throws Exception {
         addEdges(keyspace, MAX_SIZE);
 
         Grakn.factory(Grakn.DEFAULT_URI, keyspace).getGraph().showImplicitConcepts(true);
-        Grakn.factory(Grakn.DEFAULT_URI, keyspace).getGraph().graql().compute().degree().persist().execute();
+        //TODO: Remove persistence
```

Comment:
In my opinion Scale Tests should actually be externalised from the repo. Since they are not automatically run with our normal tests I would put them somewhere else. I think @mikonapoli  also has the same belief.

If we get them to run automatically (even if not frequently) then keep them.

Something to consider when talking with @sheldonkhall .